### PR TITLE
Update 01_leds for r5c power and wlan

### PR DESCRIPTION
**Pull request by IslamAlam**
add config for r5c power and wlan leds

### CODE REVIEW

File: target/linux/rockchip/armv8/base-files/etc/board.d/01_leds [hunk 1]
```diff
@@ -24,8 +24,10 @@ xunlong,orangepi-r1-plus-lts)
 	ucidef_set_led_netdev "lan" "LAN" "green:lan" "eth1"
 	;;
 friendlyarm,nanopi-r5c)
+	ucidef_set_led_default "power" "POWER" "red:power" "1"
```
Comment (1715173329) [inline target/linux/rockchip/armv8/base-files/etc/board.d/01_leds:27]: this is default setting, drop it

Comment (IslamAlam) [reply target/linux/rockchip/armv8/base-files/etc/board.d/01_leds]: Thanks for your feedback, done!

**Comment by IslamAlam** [comment]
@1715173329 It has been updated.

**Comment by 1715173329** [comment]
Merged, thanks!

**Pull request merged**